.eslintrc.json

```diff
@@ -0,0 +1,105 @@
+{
+  "root": true,
+  "env": {
+    "es6": true,
+    "node": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:import/typescript",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:@typescript-eslint/recommended-requiring-type-checking",
+    "prettier"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "project": "tsconfig.json",
+    "sourceType": "module"
+  },
+  "plugins": ["eslint-plugin-import", "header", "@typescript-eslint"],
+  "rules": {
+    "@typescript-eslint/consistent-type-assertions": "error",
+    "@typescript-eslint/no-explicit-any": "error",
+    "@typescript-eslint/no-non-null-assertion": "error",
+    "@typescript-eslint/no-unnecessary-qualifier": "error",
+    "@typescript-eslint/no-unused-expressions": "error",
+    "curly": "error",
+    "header/header": ["error", "block", [{ "pattern": "Copyright Google LLC" }]],
+    "import/first": "error",
+    "import/newline-after-import": "error",
+    "import/no-absolute-path": "error",
+    "import/no-duplicates": "error",
+    "import/no-extraneous-dependencies": ["error", { "devDependencies": false }],
+    "import/no-unassigned-import": ["error", { "allow": ["symbol-observable"] }],
+    "import/order": [
+      "error",
+      {
+        "alphabetize": { "order": "asc" },
+        "groups": [["builtin", "external"], "parent", "sibling", "index"]
+      }
+    ],
+    "max-len": [
```

Comment:
Drive by comment, do we need this, since line length is enforced via prettier.

Reply:
There are cases where a long line cannot be split and will be retained.  This rule complements the formatter by ensuring that such lines are within reason.  Some examples include regular expressions, strings, and certain comments (URLs, etc.).